build.sbt

```diff
@@ -129,8 +129,8 @@ lazy val `pekko-connectors` = project
     crossScalaVersions := List() // workaround for https://github.com/sbt/sbt/issues/3465
   )
 
-addCommandAlias("applyCodeStyle", ";scalafmtAll; scalafmtSbt; javafmtAll")
-addCommandAlias("checkCodeStyle", ";scalafmtCheckAll; scalafmtSbtCheck; javafmtCheckAll")
+addCommandAlias("applyCodeStyle", ";scalafmtAll; scalafmtSbt; javafmtAll; +headerCheckAll")
+addCommandAlias("checkCodeStyle", "+headerCreateAll; ;scalafmtCheckAll; scalafmtSbtCheck; javafmtCheckAll")
```

Comment:
+headerCheckAll

Reply:
@mdedetrich you need still update the `+headerCreateAll` to `+headerCheckAll`

Reply:
Ah I see, missed it doing it now

Reply:
Fixed and pushed.